src/Hal/Application/Config/File/ConfigFileReaderJson.php

```diff
@@ -72,6 +72,13 @@ public function read(Config $config)
             $config->set('git', $jsonData['plugins']['git']['binary']);
         }
 
+        // retrocompatibility with typo in documentation
+        // see https://github.com/phpmetrics/PhpMetrics/issues/441
+        // file -> report
+        if (isset($jsonData['plugins'], $jsonData['plugins']['junit'], $jsonData['plugins']['junit']["file"])) {
```

Comment:
As `isset` is already dealing with possible unset variable, you can directly use `$jsonData['plugins']['junit']["file"]`

```suggestion
        if (isset($jsonData['plugins']['junit']['file'])) {
```